src/app/lists/edit_modal.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
```

Comment:
The file name is in snake_case but the others are done in camelCase. Is there some sort convention? I'm curious!